### PR TITLE
x11-misc/xsensors: EAPI bump 2 -> 6, rename configure.in to .ac

### PR DESCRIPTION
**Pull request by Moiman**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

File: x11-misc/xsensors/xsensors-0.70-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+IUSE=""
+
+RDEPEND="x11-libs/gtk+:2
+	>=sys-apps/lm_sensors-3"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gtk220.patch
+	eapply_user
```
Comment (SoapGentoo) [inline x11-misc/xsensors/xsensors-0.70-r1.ebuild]: stick patch in `PATCHES` array, then replace these two lines by `default`

File: x11-misc/xsensors/xsensors-0.70-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+	mv configure.in configure.ac || die #426262
+
+	eautoreconf
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die
+	dodoc AUTHORS ChangeLog README TODO
+}
```
Comment (SoapGentoo) [inline x11-misc/xsensors/xsensors-0.70-r1.ebuild]: nix this entire `src_install`, all of it should be in `default_src_install` of EAPI 6

File: x11-misc/xsensors/xsensors-0.70-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools eutils
```
Comment (SoapGentoo) [inline x11-misc/xsensors/xsensors-0.70-r1.ebuild]: don't need `eutils` anymore

**Comment by Moiman** [comment]
Thanks for all the reviews. I learned a lot. Thanks!

**Comment by SoapGentoo** [comment]
@Moiman thanks for the help in modernising the ecosystem, keep the PRs coming! :smile:

**Pull request closed without merging**